code/index.html

```diff
@@ -1,22 +1,119 @@
 <!DOCTYPE html>
+<html lang="eng"></html>
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Project 2: Noemi's Music Magazine</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Amiri&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header> 
+      <h1>Noemi's Music Magazine <img src="./images/logo.jpg" alt="Logo" height="40px"> </h1>
+      <nav>
+        <a href="../index.html">Home</a>
+        <a>Concerts</a>
+        <a>Releases</a>
+        <a>About</a>
+      </nav>
     </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+    
+    <section class="main-article">
+      <img class="img-main-artist" src ="./images/priya.jpg" alt="Priya Ragu">
+        <div class="big-news">
+          <h2>Priya Ragu</h2>
+            <p class="author-main-article">by Noemi, 01.01.2021</p>
+            <p class="main-article-text">
+              The child of Sri Lankan parents, Ragu would sing in Tamil at family functions and made her stamp on the Zurich music scene from an early age. She just never thought her music would be heard by millions.
+              Ragu’s slick production is all down to her rapper brother Japhna Gold, proving it pays to keep it in the family. Together the duo have developed a signature sound, blending raspy vocals with an electro-pop beat that often nods to their South Indian heritage.
+              Debut single Good Love 2.0 is an absolute banger, guaranteed to get stuck in your head. If you haven’t already heard it played via Annie Mac on Radio 1 or on Radio 1Xtra earlier this year, you no doubt will have caught it on loop while playing FIFA 21.
+              For now, Ragu is working on getting her tracks out into the world and gearing up for the day she can finally perform live. ​“I just hope that we can tour, she says. ​“We just created a band, so it would be dope if we could play live again and create memories, you know.”
+              Listen to Good Love 2.0 below and get your 100% fill with Priya Ragu.</p>
```

Comment:
These paragraphs are siblings to the `h2` and should therefore not have any indentation